packages/flutter/test_fixes/material/material.dart

```diff
@@ -306,4 +306,12 @@ void main() {
 
   final PlatformMenuBar platformMenuBar = PlatformMenuBar(menus: <PlatformMenuItem>[], body: const SizedBox());
   final Widget bodyValue = platformMenuBar.body;
+
+  // Changes made in https://github.com/flutter/flutter/pull/154972
+  final Object outlineBorder = MaterialStateOutlineInputBorder.resolveWith(
```

Comment:
nit: Instead of `Object`s, can these be `InputBoder`s?

Reply:
Sure thing!